src/openms/include/OpenMS/ANALYSIS/QUANTITATION/IsotopeLabelingMDVs.h

```diff
@@ -0,0 +1,204 @@
+// --------------------------------------------------------------------------
+//                   OpenMS -- Open-Source Mass Spectrometry
+// --------------------------------------------------------------------------
+// Copyright The OpenMS Team -- Eberhard Karls University Tuebingen,
+// ETH Zurich, and Freie Universitaet Berlin 2002-2020.
+//
+// This software is released under a three-clause BSD license:
+//  * Redistributions of source code must retain the above copyright
+//    notice, this list of conditions and the following disclaimer.
+//  * Redistributions in binary form must reproduce the above copyright
+//    notice, this list of conditions and the following disclaimer in the
+//    documentation and/or other materials provided with the distribution.
+//  * Neither the name of any author or any participating institution
+//    may be used to endorse or promote products derived from this software
+//    without specific prior written permission.
+// For a full list of authors, refer to the file AUTHORS.
+// --------------------------------------------------------------------------
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL ANY OF THE AUTHORS OR THE CONTRIBUTING
+// INSTITUTIONS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+// OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+// WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+// OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+// ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+// --------------------------------------------------------------------------
+// $Maintainer: Ahmed Khalil $
+// $Authors: Ahmed Khalil $
+// --------------------------------------------------------------------------
+
+#pragma once
+
+#include <OpenMS/config.h>
+
+//Kernal classes
+#include <OpenMS/DATASTRUCTURES/DefaultParamHandler.h>
+#include <OpenMS/KERNEL/FeatureMap.h>
+#include <OpenMS/KERNEL/Feature.h>
+
+//Standard library
```

Comment:
check for unneded includes

Reply:
- `CoarseIsotopePatternGenerator.h` and `IsotopeDistribution.h` are used to calculate fragment isotopomer theoretical values. 
- `OpenMS/DATASTRUCTURES/Matrix.h` is used for operations on correction matrices. 
- `Feature.h` and `FeatureMap.h` are used for FeatureMap related operations.
- `cstddef.h` is used for `size_t`.
- `cmath.h` is used for `std::abs`.
- `unordered_map` is used as storage container for correction matrices.
- `algorithm` is used for `std::max`.
- `Eigen/Dense` is used for Eigen Matrix : `MatrixXf`.

Reply:
I think what Timo meant was: check if you need the includes in the header. Most of them can actually be moved into the cpp since they are only used there.
Others, you can forward declare, e.g. 
```
class FeatureMap;
class Feature;
```

If i'm not mistaken, you only need `Matrix.h`